### ISSUE TITLE
Node.js example

### ISSUE THREAD

**Issue by piranna** [redux-offline/redux-offline#310]
Last weekend I did a [proof-of-concept](https://github.com/piranna/redux-offline-Node.js-PoC) about how to use redux-offline in a Node.js CLI app, maybe it can be useful as an example about how to use it outside React or React Native environments... :-) The example create actions with a timer and they get print on console or queued by detecting network status, but it would be possible to do it in any other context, for example looking if a filesystem is mounted.

This also lead me to think, does it makes sense to extract redux-offline core functionality (mostly just only the queue processing) to an independent `core` module? There are cases like this that support by default of network requests is not needed...

**Comment by calumpeak** [comment]
@piranna This is stuff we're taking a look at with V3 :) To see if there are any useful separation of concerns that might be useful and to allow a bit more 'plug n play' functionality with different libraries etc.

**Comment by piranna** [comment]
Fine! Can you point me where the discussion of v3 is about? :-)

**Comment by sorodrigo** [comment]
Could you pass me your email piranna?

**Comment by piranna** [comment]
> Could you pass me your email piranna?

Yes, my professional account is jesus.leganes.combarro@gmail.com
